portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,11 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ["Custard, good. Jam, good. Meat, good.", 
+        "Joey doesn't share food!", 
+        "I don't like it when people take food off of my plate, okay?", 
+        "The fridge broke so I had to eat everything", 
+        "That's a great story. Can I eat it?"];
```

Comment:
nit: please use single quotes (') instead of double quotes (") for javascript string literals

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes